genesyscloud/util/testrunner/testrunner.go

```diff
@@ -17,7 +17,7 @@ const (
 )
 
 func GetTestDataPath(elem ...string) string {
-	basePath := filepath.Join("..", "test", "data")
+	basePath := filepath.Join("../../", "test", "data")
```

Comment:
Is this function used anywhere other than user prompts? If so changing this could break tests

Reply:
apparently it is being used. ill change it.